sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```
@@ -188,53 +228,26 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
-      if (pluginResource == null) {
-        throw CudfVersionMismatchException(s"Could not find properties file " +
-          s"$pluginPropertiesFileName in the RAPIDS Accelerator jar. Cannot verify cudf " +
-          s"version compatibility with RAPIDS Accelerator version.")
-      }
-      props.load(pluginResource)
-
-      val pluginCudfVersion = props.get("cudf_version")
-      if (pluginCudfVersion == null) {
-        throw CudfVersionMismatchException(s"Property name `cudf_version` not found in" +
-          s" $pluginPropertiesFileName file.")
+      val cudfVersion = RapidsPluginUtils.getClasspathCudfVersion
+      val pluginProps = RapidsPluginUtils.getPluginProps
+      val expectedCudfVersion = Option(pluginProps.get("cudf_version")).map(_.toString).getOrElse {
+        throw CudfVersionMismatchException("Could not find cudf version in plugin properties file")
       }
-      val expectedCudfVersion = pluginCudfVersion.toString
       // compare cudf version in the classpath with the cudf version expected by plugin
       if (!RapidsExecutorPlugin.cudfVersionSatisfied(expectedCudfVersion, cudfVersion)) {
-        throw CudfVersionMismatchException(s"Cudf version in the classpath is different. " +
-          s"Found $cudfVersion, RAPIDS Accelerator expects $expectedCudfVersion")
+        throw CudfVersionMismatchException(s"Found cudf version $cudfVersion, RAPIDS Accelerator " +
+            s"expects $expectedCudfVersion")
       }
+      val pluginVersion = Option(pluginProps.get("version")).map(_.toString).getOrElse {
+        throw new PluginException("No `version` property in plugin properties file")
+      }
+      logInfo(s"RAPIDS Accelerator $pluginVersion using cudf $cudfVersion")
     } catch {
-      case x: CudfVersionMismatchException if conf.cudfVersionOverride =>
-        logWarning(s"${x.errorMsg}")
+      case x: PluginException if conf.cudfVersionOverride =>
+        logWarning(s"${x.getMessage}")
```

can we add  some language here about exception being ignored due to cudfVersionOverride being on.